module/profile/helper.js

```diff
@@ -0,0 +1,109 @@
+const axios = require('axios')
+const entityFind = require('../../generics/services/entity-management')
```

Comment:
@MallanagoudaB not the correct way to import the file. Also the variable name is improper
sample : const entitiesService = require(GENERICS_FILES_PATH + '/services/entity-management')